runtime/j9vm/jvm.c

```diff
@@ -806,10 +834,10 @@ static J9StringBuffer* jvmBufferEnsure(J9StringBuffer* buffer, UDATA len) {
 
 	if (len > buffer->remaining) {
 		UDATA newSize = len > MIN_GROWTH ? len : MIN_GROWTH;
-		J9StringBuffer* new = (J9StringBuffer*) malloc( strlen((const char*)buffer->data) + newSize + sizeof(UDATA) + 1 );
-		if (new) {
+		J9StringBuffer *new = (J9StringBuffer *)malloc(strlen((const char *)buffer->data) + newSize + sizeof(UDATA) + 1);
```

Comment:
While you're here, please update this to use a better name than `new` (making it easier to switch to C++ in the future).